frontend/src/components/AccessGame/GameState.vue

```diff
@@ -13,47 +13,49 @@
       </div>
     </template>
     <template v-else>
-      <div v-if="game.status === 'created'" class="card">
+      <div v-if="game && game.status === 'created'" class="card">
         <div class="card-body">
           <h6>Invite other players to join</h6>
-          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': player.isHost }">{{ game.id }}</div>
-          <p v-if="players.length > 1" class="mt-3 mb-0">
+          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': !!(player || {}).isHost }">
+            {{ (game || {}).id }}
+          </div>
+          <p v-if="players && players.length > 1" class="mt-3 mb-0">
             Other players in this game:
             <span
-              v-for="(gamePlayer, index) in players"
-              v-if="gamePlayer.nickname !== player.nickname"
+              v-for="(gamePlayer, index) in (players || [])"
+              v-if="player && gamePlayer.nickname !== player.nickname"
               :key="gamePlayer.id"
-              >{{ gamePlayer.nickname }} <template v-if="index < players.length - 1">, </template></span
+              >{{ gamePlayer.nickname }} <template v-if="players && index < players.length - 1">, </template></span
             >
           </p>
         </div>
       </div>
       <button
-        v-if="player.isHost && !rounds.length && players.length > 2"
+        v-if="player && rounds && player.isHost && !rounds.length && players && players.length > 2"
```

Comment:
Rather long, could be a computed property.

Reply:
I propose to fix them in #99.

Reply:
👍